code_share/code_share/settings.py

```diff
@@ -13,7 +14,7 @@
 SECRET_KEY = '6*yy_^z%min*=(w9rjxj&f*fgl6daq3j)_52my8i)u5hp)9g6#'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = config('DEBUG'),
```

Comment:
In that case docs have to be updated too specify what all are needed in `.env` file. Currently we are using a `local_settings.py` file for storing `SECRET_KEY` and other `AUTH_KEYS`, maybe we can use that to specify `DEBUG` status instead of having an extra dependency and a file. What say?